docs/requirements.txt

```diff
@@ -3,6 +3,7 @@ sphinx==2.2.2
 ipykernel
 nbsphinx
 cython>=0.20
+numba
 numpy>=1.9
 sympy>=1.5.1
 repoze.lru>=0.7
```

Comment:
You will need to add scipy here for the docs to build, now that Scipy is not mocked out!

Reply:
```suggestion
repoze.lru>=0.7
scipy>=1.2.1
```